tests/factory_fixtures/notes.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from models.notes import NotesModel
+
+@pytest.fixture
+def note_attributes():
+    def _note_attributes(text, user, ticket):
+        return {
+            'text': text,
+            'user': user.id,
+            'ticketid': ticket.id
+        }
+    yield _note_attributes
+
+@pytest.fixture
+def create_note(note_attributes, create_join_staff):
+    class SimpleTicket:
```

Comment:
Technically, if we're going to use Fakes. Then these should have tests too, however there is not really anything to test in these fakes, and that in itself raises some flags for me. When a Note factory is used then it is going to be referring to a User and Ticket that are not true representations of what they're doubling. And that could cause issues for a developer when they need the `note.user` or the `note.ticket` when testing. I realize that it is possible to pass in a real user and ticket with this structure, however it is common practice for a factory to return real objects by default.

Reply:
I tried rebasing to so that I could use the ticket fixture. But I get a fixture 'faker' not found error. This seems to be due to the 'user_attributes' function in the user factory fixtures. Am I missing something?

Reply:
Rerunning ```pipenv install -d``` fixed the issue.